### ISSUE TITLE
[Request]: tlrc

### ISSUE THREAD

**Issue by nataziel** [ScoopInstaller/Main#5733]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

tlrc

### Description

a rust client for tldr pages

### Homepage

https://github.com/tldr-pages/tlrc

### Download Link(s)

https://github.com/tldr-pages/tlrc/releases/download/v1.9.1/tlrc-v1.9.1-x86_64-pc-windows-msvc.zip

### Some Indication of Popularity/Repute

tldr-pages repo has 48k stars, so there's clearly a lot of people who use it. The only app in the main bucket that uses tldr-pages is unmaintained. I have opened a PR for this issue
